variables.tf

```diff
@@ -170,10 +170,11 @@ variable "control_tower_account_ids" {
 
 variable "datadog" {
```

Comment:
variable "datadog" {
  type = object({
    api_key                 = string
    enable_integration      = bool
    install_log_forwarder   = optional(bool,false)
    log_collection_services = optional(list(string), [])
    site_url                = string
  })
  default     = null
  description = "Datadog integration options for the core accounts"
}